packages/m/miaoyan.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "miaoyan",
+  "description": "Lightweight Markdown app to help you write great sentences.",
+  "filename": "package.json",
+  "keywords": [
+    "MiaoYan",
+    "Markdown"
+  ],
+  "homepage": "https://miaoyan.app/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tw93/MiaoYan.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tw93/MiaoYan.git",
+    "fileMap": [
+      {
+        "basePath": "Resources",
+        "files": [
+          "**/*.+(js|css|ttf)"
```

Comment:
Can we avoid using a globstar here please, and instead specify which directories assets should be pulled from?